cmd/kueuectl/app/passthrough/passthrough.go

```diff
@@ -23,55 +23,92 @@ import (
 	"syscall"
 
 	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+
+	"sigs.k8s.io/kueue/cmd/kueuectl/app/delete"
+	"sigs.k8s.io/kueue/cmd/kueuectl/app/util"
 )
 
-type passThroughType struct {
+type passThroughCommand struct {
 	name    string
+	short   string
 	aliases []string
 }
```

Comment:
I don't like this "type reuse", lets keep `passThroughType`  and define a different one for command if necessary (but for mentioning that it's a pass to kubectl is sufficient)/

Reply:
Reverted.

Reply:
> and define a different one for command if necessary (but for mentioning that it's a pass to kubectl is sufficient)/

Ah sorry missed it.